08-projects.Rmd

```diff
@@ -6133,6 +6133,31 @@ The following questions will use the dataset found in Scholar:
 
 A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/movies-and-tv/imdb.db).
 
+**Important note:** For this project you will use SQLite to access the data. To connect to the database, copy and paste the following before your solutions in your .Rmd:
```

Comment:
This note is based on the one in Project 12. I removed the requirement of using SQL code chunk so that it is okay for some students to use R functions like `dbGetQuery`.

Reply:
Thanks!